### DISCUSSION TITLE
PCE games wont show

### DISCUSSION THREAD

**Discussion by mchenier** [eduardofilo/RG350_adam_image#138] [SimpleMenu topics]
I see every other system in the menu that I have roms for but the menu for PCE just don't show up when there is rom in the PCE folder and also I added the BIOS syscard*

I am currently on 1.2. (1.31 didn't work for me)

**Comment by eduardofilo** [comment]
For the systems to appear in SimpleMenu, there must be at least one ROM with the configured extensions. For PCE in 1.2 the valid extensions were:

```
romExts = .pce,.tg16,.cue,.7z
```

Check if ROMs use one of those extensions.

**Reply by mchenier** [reply]
Yes that was the problem. When I checked the doc it says it was accepting .zip file but it must be for 1.3 and above I suppose.

Thanks.

**Reply by eduardofilo** [reply]
That's right. It was incorporated in version 1.3.
Anyway you can add the extension in the configuration file `/media/data/local/home/.simplemenu/section_groups/consoles.ini` changing the `romExts` parameter in that system to this:

```
[PC ENGINE]
execs = /media/data/apps/retroarch_rg350_PCE.opk
romDirs = /media/sdcard/roms/PCE/
romExts = .pce,.tg16,.cue,.zip,.7z
scaling = 0
```
